### PR TITLE
release 0.7.0 - SARSCov2 Illumina Full support In Terra

### PR DESCRIPTION
**Pull request by ehigham**

### CODE REVIEW

File: api/src/wfl/module/covid.clj [hunk 1]
```diff
@@ -0,0 +1,1052 @@
...
+                      {:fromOutputs fromOutputs :workflow workflow}
+                      cause)))))
+
+(defn ^:private entity-exists? [workspace entity]
+  "Test if the `entity` exists in the `workspace`."
+  (try
+    (firecloud/get-entity workspace entity)
+    (catch ExceptionInfo ex
+      (when (not= (-> ex ex-data :status) 404)
+        (throw ex)))))
```
Comment (tbl3rd) [inline api/src/wfl/module/covid.clj:1018]: The docstring is in the wrong place.

**Comment by okotsopoulos** [comment]
Is there a changelog that's automatically generated as part of a release?

**Comment by ehigham** [comment]
> Is there a changelog that's automatically generated as part of a release?
> 
> ETA: ah, step 5, guessing you will run it once your last fix has been cherry-picked in.
> https://broadinstitute.github.io/wfl/dev-release/

yep!

**Review by okotsopoulos — APPROVED** [review]

**Review by tbl3rd — APPROVED** [review]
I skipped much of the test code and all of the docs in the interest of time.

**Comment by ehigham** [comment]
> I skipped much of the test code and all of the docs in the interest of time.

The docs are pretty lacking. I think a patch release for documentation would be worthwhile and as part of understanding our technical debt.

**Comment by okotsopoulos** [comment]
> > I skipped much of the test code and all of the docs in the interest of time.
> 
> The docs are pretty lacking. I think a patch release for documentation would be worthwhile and as part of understanding our technical debt.

Strong agreement here!  I'd include docstrings in with docs.

**Review by rexwangcc — APPROVED** [review]
LGTM!

**Pull request merged**